src/stylesheets/common/_cta.scss

```diff
@@ -17,9 +17,7 @@
 .cta-text {
   // Match text size and line height to button.
```

Comment:
if we don't need font-size and line-height here anymore please remove this comment